server/server.go

```diff
@@ -319,10 +295,16 @@ func (s *ArgoRolloutsServer) WatchRolloutInfos(q *rollout.RolloutInfoListQuery,
 	ctx := context.Background()
```

Comment:
`ws.Context() ` should be used instead of `context.Background()`

Reply:
Fixed